python/mooseutils/PostprocessorReader.py

```diff
@@ -0,0 +1,52 @@
+from MooseDataFrame import MooseDataFrame
+import tools
+
+class PostprocessorReader(MooseDataFrame):
+    """
+    An extension to the MooseDataFrame to provide functionality mirroring that of the VectorPostprocessorReader.
+
+    Args:
+        filename[str]: The csv file to read.
+    """
+
+    def __init__(self, filename, **kwargs):
+        super(PostprocessorReader, self).__init__(filename, **kwargs)
+
+    def __call__(self, keys, time=None, warning=True, **kwargs):
+        """
+
```

Comment:
Return the data for the supplied keys.

Args:
    keys[list]: The names of the postprocessors to return.
    time: Required for consistent calls in Peacock, but not used in general.
    warning: When true (default) an error is produced if the users tries to use 'time' option, which does nothing.